src/_guides/language/effective-dart/usage.md

```diff
@@ -1079,12 +1078,7 @@ class Point {
 }
 {% endprettify %}
 
-<!-- TODO(rnystrom): Uncomment this when all Dart tools support it. -->
-<!--
-
-**TODO: Use "###" for header. Removed so this doesn't go into TOC.**
-
-DON'T use `new`.
+### DON'T use `new`.
 
 Dart 2 makes the `new` keyword optional. Even in Dart 1, its meaning was never
```

Comment:
Why no "bad" example?

Reply:
(I could add it for you, if you just want to be done with this.)

Reply:
Done.